src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -967,6 +973,7 @@ def make_catalog(ws, sql_backend, make_random) -> Generator[Callable[..., Catalo
     def create() -> CatalogInfo:
         name = f"ucx_C{make_random(4)}".lower()
         sql_backend.execute(f"CREATE CATALOG {name}")
+        sql_backend.execute(f"ALTER CATALOG {name} SET TAGS ('RemoveAfter' = '{get_test_purge_time()}')'")
```

Comment:
afaik, we cannot retrieve tags via rest api - see https://github.com/databrickslabs/watchdog/blob/main/scan/objects/uc.go#L68-L76